tests/conftest.py

```diff
@@ -28,6 +28,17 @@
                                     PerformanceSignature)
 from treeherder.services.pulse.exchange import get_exchange
 
+APIClient = None
+try:
+    # this is part of circular import but
+    # it imports same-named modules in the correct order
+    from rest_framework.test import APIClient  # noqa: F401
+except ImportError:
```

Comment:
When do you encounter this problem?

Reply:
Fixed.   This was the result of using later versions on my local machine instead of the hashes in `requirements`